MultiTable/src/main.js

```diff
@@ -30,7 +30,8 @@ const template = {
 const RealtimeChangeHandler = require('./RealtimeChangeHandler');
 const SubjectGroups = require('./SubjectGroups');
 
-const EVENT_CATEGORY = 'Lens - Multitable';
+const VERSION = require('../lens.json').version;
```

Comment:
see above comment for the updated zip. 
Here's what the category looks like, with lens version
![screen shot 2017-08-07 at 10 36 28 am](https://user-images.githubusercontent.com/17040750/29038432-793b3fdc-7b5c-11e7-8c0e-e86fc99dac07.png)